.github/workflows/retrigger-rtd.yml

```diff
@@ -0,0 +1,25 @@
+name: Retrigger RTD build on PR comment
+
+on:
+  issue_comment:
+    types: [created]
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  retrigger-rtd:
+    if: github.event.issue.pull_request &&
+        (contains(github.event.comment.body, 'jenkins test docs') || 
+         contains(github.event.comment.body, 'jenkins rebuild docs') || 
+         contains(github.event.comment.body, 'jenkins retrigger docs'))
```

Comment:
It's a tad bit confusing to prefix these with "jenkins" but I realize that's for matching historical commands. But since you're allowing different variations maybe keep the jenkins prefix for backwards compat. but have non-legacy variations that are less misleading wrt jenkins not being involved :-)

Reply:
Can't win 😆 

https://github.com/ceph/ceph/pull/61938#discussion_r1964385093

Reply:
lol. in my defense I did say to keep the old ones for compatibility :-)
Plus, the file appears new in this PR so I didn't know this had been discussed previously.

Reply:
I know.  Just gentle ribbing.  🙂